uportal-war/src/main/webapp/media/skins/muniversality/common/common_skin.xml

```diff
@@ -54,8 +54,8 @@
     <js included="plain" resource="true">/rs/underscore/1.5.2/underscore-1.5.2.js</js>
     <js included="aggregated" resource="true">/rs/underscore/1.5.2/underscore-1.5.2.min.js</js>
 
-    <js included="plain" resource="true">/rs/backbone/1.0.0/backbone-1.0.0.js</js>
-    <js included="aggregated" resource="true">/rs/backbone/1.0.0/backbone-1.0.0.min.js</js>
+    <js included="plain" resource="true">/rs/backbone/1.3.3/backbone-1.3.3.js</js>
+    <js included="aggregated" resource="true">/rs/backbone/1.3.3/backbone-1.3.3.min.js</js>
```

Comment:
Requires a new build of `resource-server-content` and must be accompanied with a version increment in the root `pom.xml` file.  I will take care of it.